lib/msal-angular/dist/msal-angular.js

```diff
@@ -30433,6 +30433,10 @@ var MsalGuard = /** @class */ (function () {
             return currentAbsoluteUrl.substring(0, index);
         }
     };
+    MsalGuard.prototype.isObjectEmpty = function (obj) {
```

Comment:
is this duplicated somewhere else? I remember commenting on it in @sameerag  pr somewhere

Reply:
nm this is dist